client/coral-embed-stream/src/components/Comment.js

```diff
@@ -447,6 +447,14 @@ export default class Comment extends React.Component {
 
               {isStaff(comment.tags) ? <TagLabel>Staff</TagLabel> : null}
 
+              <Slot
+                className={cn('talk-stream-comment-author-tags')}
+                fill="commentAuthorTags"
+                queryData={queryData}
+                {...slotProps}
+                inline
```

Comment:
I'd love to deprecated the `inline` property and just use `className` to pass styling into the Slot instead. wdyt?

Reply:
TBH I love `inline` it makes it so much easier to develop inline slots. Any specific reason for it?

Reply:
I'd say we should merge this so we have the recipe up and running. And we can revisit this later and all together in another PR.

Reply:
Yeah it's not a big issue, just wanting to hear your thoughts, and trying to have a clear concept for the `inline` property, which I don't have.